api/stream.go

```diff
@@ -36,15 +40,17 @@ func NewStreamAPI(
 	receipts storage.ReceiptIndexer,
 	accounts storage.AccountIndexer,
 	blocksBroadcaster *engine.Broadcaster,
+	transactionsBroadcaster *engine.Broadcaster,
 ) *StreamAPI {
 	return &StreamAPI{
-		logger:            logger,
-		config:            config,
-		blocks:            blocks,
-		transactions:      transactions,
-		receipts:          receipts,
-		accounts:          accounts,
-		blocksBroadcaster: blocksBroadcaster,
+		logger:                  logger,
+		config:                  config,
+		blocks:                  blocks,
+		transactions:            transactions,
+		receipts:                receipts,
+		accounts:                accounts,
```

Comment:
Do we have a use-case for the `accounts` value being used here? It does not seem to be used anywhere in this `StreamAPI` type :thinking: ? Or is it needed for a follow-up PR?